op-chain-ops/script/script.go

```diff
@@ -54,6 +55,33 @@ type CallFrame struct {
 	Prank *Prank
 }
 
+type Broadcast struct {
+	From     common.Address
+	To       common.Address
+	Calldata []byte
+	Value    *big.Int
+}
+
+var zero = big.NewInt(0)
```

Comment:
There is `common.Big0` to do comparisons against

Reply:
Nice, will refactor.